sycl/test-e2e/ESIMD/accessor_global.cpp

```diff
@@ -0,0 +1,121 @@
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: gpu-intel-gen9 && windows
+//
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies usage of accessor methods operator[] and get_pointer().
+
+#include "esimd_test_utils.hpp"
+
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+#include <iostream>
+
+using namespace sycl;
+using namespace sycl::ext::intel::esimd;
+
+constexpr int VL = 16;
+
+template <typename T, bool TestSubscript>
+bool test(queue Q, uint32_t LocalRange, uint32_t GlobalRange) {
+  std::cout << "Running case: T=" << esimd_test::type_name<T>()
+            << ", TestSubscript=" << TestSubscript << std::endl;
+
+  T *Tmp = new T[GlobalRange * VL];
```

Comment:
Looks dead.

Reply:
What do you mean? This is the only allocation of host memory. Please elaborate the comment.

Reply:
This was one of greatest comments.
My attempt to verify the content of Tmp on host after the device code showed a huge error in how the initial version of the patch handled accessors without -fsycl-esimd-force-stateless-mem.
The test worked as it was initially, but on device it wrote to some random memory, then read from it.

To fix that issue, I had to revert part of initial changes I did in ESIMDVerifierPass().